test/functional/content_view/version/incremental_update_test.rb

```diff
@@ -88,4 +88,21 @@
     result = run_cmd(@cmd + params)
     assert_equal(result.exit_code, 0)
   end
+
+  it "performs incremental update with packages" do
+    params = ['--content-view-version-id=5', '--packages=bla', '--lifecycle-environment-ids=1']
+
+    expect_generic_search(
+      :packages, params: {search: "name = \"bla\""}, returns: {'id' => 15}
+    ).returns(index_response([{'id' => 15}]))
```

Comment:
You can put the returns here or on line 96, but you don't need it in both places. I think the last one gets honored, so the `returns: {'id' => 15}` isn't doing anything.